api/src/main/kotlin/org/jetbrains/kotlinx/dl/api/core/layer/normalization/BatchNorm.kt

```diff
@@ -57,31 +52,27 @@ public class BatchNorm(
     public val movingMeanInitializer: Initializer = Zeros(),
     public val movingVarianceInitializer: Initializer = Ones(),
     name: String = "",
-) : Layer(name), NoGradients {
+) : Layer(name), NoGradients, ParametrizedLayer {
```

Comment:
Could we make NoGradients a subtype of Parametrized but notTrainable?

Reply:
Actually, no. There is, for example, `Concatenate` layer which is not `Parametrized` but implements `NoGradients`.